CHANGELOG.md

```diff
@@ -4,6 +4,26 @@ This changelog goes through all the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.io).
 
+## [1.12.2](https://github.com/go-gitea/gitea/releases/tag/v1.12.2) - 2020-07-11
+
+* BUGFIXES
+  * When deleting repository decrese user repositry count in cache (#11954) (#12188)
+  * Gitea commits API again returns commit summaries, not full messages (#12186) (#12187)
```

Comment:
```suggestion
  * Return full commit message instead of summary in commits API (#12186) (#12187)
```
Current wording makes it sound like we intentionally changed it back to summaries